app/assets/stylesheets/objects/_side.scss

```diff
@@ -7,15 +7,17 @@
 }
 
 .app-side__actions {
-  @include govuk-responsive-margin(3, "bottom");
-
   .govuk-button {
     width: 100%;
     margin-bottom: govuk-spacing(3);
   }
 
-  .govuk-link {
-    margin-bottom: govuk-spacing(3);
+  > .govuk-button:last-child {
```

Comment:
We generally try to avoid using child combinators as they heavily rely on markup structure, but given the fact that we're unable to be in control on how the markup looks for an action (it could be inside our outside a form) I don't see a better option here..

Reply:
Yeah I felt it wasn't ideal but only adding very specific classes seemed to be an alternative.